examples/util/base.js

```diff
@@ -253,16 +256,39 @@ const ssdlite_mobilenetv2_tflite = {
 
 const posenet = {
   modelName: 'PoseNet',
+  modelSize: '13.3MB',
+  modelFile: '../skeleton_detection/model/mobilenet_v1_101',
   inputSize: [513, 513, 3],
   preOptions: {
     mean: [127.5, 127.5, 127.5],
     std: [127.5, 127.5, 127.5],
   },
   intro: 'PoseNet is a machine learning model that allows for Real-time Human Pose Estimation which can be used to estimate either a single pose or multiple poses.',
-  paperUrl: 'https://arxiv.org/abs/1803.08225',
-  modelCDNUrl: 'https://cdn_host_and_path/'
+  paperUrl: 'https://arxiv.org/abs/1803.08225'
 };
 
+const deeplab_mobilenet_v2_513_tflite = {
+  modelName: 'deeplab_mobilenet_v2_513_tflite',
+  modelSize: '9.5MB',
+  modelFile: '../semantic_segmentation/model/deeplab_mobilenetv2_513.tflite',
+  labelsFile: '../semantic_segmentation/model/labels.txt',
+  inputSize: [513, 513, 3],
+  outputSize: [513, 513, 21],
+  intro: 'DeepLab is a state-of-art deep learning model for semantic image segmentation, where the goal is to assign semantic labels (e.g., person, dog, cat and so on) to every pixel in the input image.',
+  paperUrl: 'https://arxiv.org/abs/1802.02611'
+}
+  
+const deeplab_mobilenet_v2_513_dilated_tflite = {
+  modelName: 'deeplab_mobilenet_v2_513_dilated_tflite',
+  modelSize: '8.4MB',
+  modelFile: '../semantic_segmentation/model/deeplab_mobilenetv2_513_dilated.tflite',
+  labelsFile: '../semantic_segmentation/model/labels.txt',
+  inputSize: [513, 513, 3],
+  outputSize: [513, 513, 21],
+  intro: 'Extend DeepLabv3 to include a simple yet effective decoder module to refine the segmentation results especially along object boundaries. Furthermore, in this encoder-decoder structure one can arbitrarily control the resolution of extracted encoder features by atrous convolution to trade-off precision and runtime.',
```

Comment:
I suggest changing the description to:
> Equivalent to the model above (without `dilated` suffix) but only available on platforms that natively support atrous convolution.

Reply:
Thanks @pinzhenx , fixed in https://github.com/intel/webml-polyfill/pull/538/commits/2aa77f76375590abe7fdf53b30cd70f53b940d21